test/test_nestedtensor.py

```diff
@@ -4339,6 +4353,115 @@ def forward(self, query, value, offsets):
         self.assertTrue(torch.allclose(attn_output_eager, attn_output))
         self.assertTrue(torch.allclose(value_grad, value.grad))
 
+    @dtypes(torch.float32)
```

Comment:
do you know if this will work with a max_seqlen manually assigned in the graph? (and if so, can you add a test / or add as a followup)

Reply:
good Q! this works with manual assignment as done in FIRST (i.e. via `convert_jagged_to_nested_tensor() -> ViewNestedFromBuffer` with the metadata cache info). Also see `test_dummy_mha_with_nt` which tries to match as closely as possible to what the FIRST model does.

We should arguably support and test manual max seq len assignment for the new API. I'll address this or add a TODO

Reply:
Update: I think I'll boot handling manual assignment outside of the FIRST use case until we can have public `@property` getters / setters for min / max sequence length that work with PT2 (this is in progress)